rmgpy/rmg/main.py

```diff
@@ -1348,6 +1356,60 @@ def readMeaningfulLineJava(self, f):
         return line
     
 ################################################################################
+def processToSpeciesNetworks(obj,reactionSystem,coreSpecies):
```

Comment:
Not particularly, other than that the RMG class itself is unaware of everything but coreSpecies, but I guess making it larger and making these module level will look cleaner.  I also don't expect we'll be calling these outside of the RMG class.